lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -168,7 +168,7 @@ defmodule Membrane.ICE.Endpoint do
     availability: :on_request,
     accepted_format: _any,
     mode: :pull,
-    demand_unit: :buffers
+    demand_mode: :auto
```

Comment:
I assume that the old `mode` field is redundant? 

Also, in the previous version we were guaranteed not to receive demands until ICE connection was established. Do we know how `auto` mode will behave?

Reply:
After establishing an ICE connection `ICE.Endpoint` sends keying material in upstream `event` to the encryptor. Encryptor won't send buffers to ICE before keying material arrival